src/main/java/com/theelixrlabs/healthcare/model/Doctor.java

```diff
@@ -0,0 +1,20 @@
+package com.theelixrlabs.healthcare.model;
+
+import com.theelixrlabs.healthcare.constants.DoctorConstants;
+import lombok.Data;
+import org.springframework.data.annotation.Id;
+import org.springframework.data.mongodb.core.mapping.Document;
+
+import java.util.UUID;
+
+@Data
+@Document(collection = DoctorConstants.DOCTOR_COLLECTION_NAME)
+public class Doctor {
+    @Id
+    private String uuid = UUID.randomUUID().toString();
```

Comment:
1. Assigning the id value here is not recommended 
2. Also UUID can be used in place of String
3. variable name uuid is not correct. UUID is a type which represents id value so keep the name id itself

Reply:
1.done
2.have doubt on this.
3.done